Detectors/TPC/simulation/include/TPCSimulation/DigitCRU.h

```diff
@@ -35,7 +35,7 @@ class DigitCRU{
     
     /// Constructor
     /// \param mCRU CRU ID
-    DigitCRU(int mCRU);
+    DigitCRU(int mCRU, const CommonModeContainer &commonModeCont);
```

Comment:
Ah, you meant  DigitCRU(int mCRU, CommonModeContainer const &commonModeCont);
Right?